Resources/Xib/MiniPlayer/AudioMiniPlayer.xib

```diff
@@ -0,0 +1,170 @@
+                                </stackView>
+                            </subviews>
+                            <connections>
+                                <outletCollection property="gestureRecognizers" destination="brR-fc-lZO" appends="YES" id="Nb4-rh-jQL"/>
+                                <outletCollection property="gestureRecognizers" destination="F5d-0u-KFK" appends="YES" id="vlA-dj-cvy"/>
+                                <outletCollection property="gestureRecognizers" destination="LWT-xp-2fu" appends="YES" id="ska-dZ-uCt"/>
+                            </connections>
+                        </stackView>
+                        <stackView opaque="NO" contentMode="scaleToFill" spacing="5" translatesAutoresizingMaskIntoConstraints="NO" id="Lh6-Gu-frg" userLabel="Control Stack View">
+                            <rect key="frame" x="214" y="0.0" width="130" height="40"/>
+                            <subviews>
+                                <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="Hmp-0g-6lI" userLabel="Previous">
+                                    <rect key="frame" x="0.0" y="0.0" width="40" height="40"/>
+                                    <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                    <constraints>
+                                        <constraint firstAttribute="width" constant="40" id="Seu-Pw-atZ"/>
+                                        <constraint firstAttribute="height" constant="40" id="upi-oq-Tgr"/>
+                                        <constraint firstAttribute="width" secondItem="Hmp-0g-6lI" secondAttribute="height" multiplier="1:1" id="zAa-Io-XGm"/>
+                                    </constraints>
+                                    <state key="normal" image="MiniPrev"/>
+                                    <connections>
+                                        <action selector="handlePrevious:" destination="-1" eventType="touchUpInside" id="3MB-Hc-E2Q"/>
+                                    </connections>
+                                </button>
+                                <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="2Kp-xd-t2N" userLabel="Play">
+                                    <rect key="frame" x="45" y="0.0" width="40" height="40"/>
+                                    <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                    <state key="normal" image="MiniPause"/>
+                                    <state key="selected" image="MiniPlay"/>
```

Comment:
you shouldn't use the selected state here to visually distinguish between play and pause. This is usually a state where you display the same image in a different color to show that the user ticked it or selected it to mark an option.